### PR TITLE
EndPoint for updating public data of Organization

### PR DESCRIPTION
**Pull request by walterio123**
**Summary**

- Creation of interface Entity to dto.
- Creation of service for update entity Organization.
- Creation of the route to update the data of the Organization.
- Modification of the security so that the data update of the Organization entity is accessible only by the ADMIN role.

### CODE REVIEW

File: src/main/java/com/alkemy/ong/service/OrganizationService.java [hunk 1]
```diff
@@ -28,4 +33,15 @@ public OrganizationDto getPublicOrganization(String organizationId) {
                     "No se encontró organización con esos datos");
         }
```
Comment (anavduo) [inline src/main/java/com/alkemy/ong/service/OrganizationService.java:34]: The comment in the Update method is in English and this one in Spanish for a reason?

File: src/main/java/com/alkemy/ong/service/OrganizationService.java [hunk 2]
```diff
@@ -1,19 +1,24 @@
...
+import lombok.var;
 import org.springframework.beans.factory.annotation.Autowired;
 import org.springframework.http.HttpStatus;
 import org.springframework.stereotype.Service;
 import org.springframework.web.server.ResponseStatusException;
 import java.util.Optional;
 
 @Service
+@RequiredArgsConstructor
 public class OrganizationService {
```
Comment (anavduo) [inline src/main/java/com/alkemy/ong/service/OrganizationService.java:19]: Is the OrganizationService implemented?

File: src/main/java/com/alkemy/ong/service/OrganizationService.java (hunk 1 above)
Comment (walterio123) [reply src/main/java/com/alkemy/ong/service/OrganizationService.java:34]: The first comment does not correspond to my code, that's why the difference.

File: src/main/java/com/alkemy/ong/service/OrganizationService.java (hunk 2 above)
Comment (walterio123) [reply src/main/java/com/alkemy/ong/service/OrganizationService.java:18]: Good point, fixed.

**Review by juan1977lopez — APPROVED** [review]
Approved

**Pull request merged**